### ISSUE TITLE
Puppetry on Pop OS 21.04

### ISSUE THREAD

**Issue by ashaikh** [dsheiko/puppetry#105]
Install errors when using the snap package to install puppetry on pop os 21.04

Failed to load module "appmenu-gtk-module"
bind-WARNING **: 01:56:00.647: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-HqMFh4ZYEM: No such file or directory
Suite file /home/username/Documents/test/.snippets.json not found.
Renderer process: node_modules not found Error: ENOENT: no such file or directory, lstat '/home/username/snap/puppetry/60/.config/puppetry/.runtime-test/node_modules'

**Comment by rsaleri** [comment]
did you try installing the module?
`sudo apt-get install appmenu-gtk2-module appmenu-gtk3-module`
